doc/usecases/elektra_web/UC_remove_cluster.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Remove cluster
```

Comment:
I thought the browser is directly connected to a clusterd? Maybe you need an alternative scenario if the user tries to do that?